client/my-sites/earn/ads/index.jsx

```diff
@@ -0,0 +1,392 @@
+					}
+				>
+					<img
+						src="/calypso/images/illustrations/waitTime.svg"
+						width="170"
+						height="143"
+						alt="WordPress logo"
+					/>
+				</ActionCard>
+			</div>
+		);
+	}
+
+	renderBannerApplicationApprove() {
+		return (
+			<div>
+				<ActionCard
+					headerText={ 'Welcome to WordAds!' }
+					mainText={
+						'We’ll work behind the scenes to maximize your earning potential and monitor ad quality. On your end, the more you grow your audience and increase your pageviews, the more you can expect to earn.'
+					}
+				>
+					<img
+						src="/calypso/images/illustrations/wordAds.svg"
+						width="170"
+						height="143"
+						alt="WordPress logo"
+					/>
+				</ActionCard>
+			</div>
+		);
+	}
+
+	renderBannerApplicationDenyBrandSafety() {
+		return (
+			<div>
+				<ActionCard
+					headerText={ "Your content isn't a good fit for ads." }
+					mainText={
+						"Our advertisers have strict requirements for the type of content they're willing to monetize."
+					}
+				>
+					<img
+						src="/calypso/images/illustrations/security-issue.svg"
+						width="170"
+						height="143"
+						alt="WordPress logo"
+					/>
+				</ActionCard>
+			</div>
+		);
+	}
+
+	renderBannerApplicationDenyTraffic() {
+		return (
+			<div>
+				<ActionCard
+					headerText={ 'You need more traffic to join WordAds.' }
+					mainText={
+						"Keep building your audience, and as soon as you have enough traffic, we'll approve your application."
+					}
+				>
+					<img
+						src="/calypso/images/illustrations/whoops.svg"
+						width="170"
+						height="143"
+						alt="WordPress logo"
+					/>
+				</ActionCard>
+				<br />
+
+				<Banner
+					title="Upgrade to a Premium Plan to bypass traffic restrictions."
+					description="Sites with a Premium Plan can run ads, regardless of how much traffic they have."
+					callToAction="Upgrade to Premium"
+					disableHref
+					event="track_event"
+					href="https://wordpress.com/"
+					icon="star"
+					prices={ [ 10.99, 9.99 ] }
+				/>
+			</div>
+		);
+	}
+
+	statsNotice() {
+		return (
+			<div>
+				<ActionCard
+					buttonPrimary
+					buttonHref="https://wordpress.com"
+					headerText="Monitor Your Ads Performance"
+					mainText="In the stats section, you can monitor the daily performance of the ads running on your site. After each month is finalized, you'll find a revenue report below (usually around 15 days after the close of the month)."
+					buttonText="View Daily Stats"
+				/>
+			</div>
+		);
+	}
+
+	renderUpsell() {
+		const { translate, adsProgramName } = this.props;
+		return (
+			<UpgradeNudgeExpanded
+				plan={ this.isJetpack ? PLAN_JETPACK_PREMIUM : PLAN_PREMIUM }
+				title={ translate( 'Upgrade to the Premium plan and start earning' ) }
+				subtitle={ translate(
+					"By upgrading to the Premium plan, you'll be able to monetize your site through the %(program) program.",
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'By upgrading to the Premium plan, you'll be able to monetize your site through the WordAds program.' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/renovate/wordads-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).